app/controllers/application_controller/performance.rb

```diff
@@ -202,8 +206,13 @@ def perf_set_or_fix_dates(options, allow_interval_override = true)
     edate_daily = edate.hour < 23 ? edate - 1.day : edate
     options[:edate_daily] = edate_daily
     # check if Daily report was manualy chosen in UI
-    if allow_interval_override
-      options[:typ] = "Hourly" if options[:typ] == "Daily" && edate_daily < sdate_daily
+    if options[:typ] == "Daily" && edate_daily < sdate_daily
+      options[:no_daily] = true
+      if allow_interval_override
+        options[:typ] = "Hourly"
```

Comment:
@bronaghs can be changed to be on single line: options[:typ] = "Hourly" if allow_interval_override